### ISSUE TITLE
Not working

### ISSUE THREAD

**Issue by naviocean** [Majestcx/OpenLoad-Downloader#5]
Hi,
I've tested your code but it did not work. It returned an error.
Please check and fix asap.
Thanks

**Comment by Sarfroz** [comment]
yeah they have encoded the stream url too now. Something like this 3. T·ÊÅﾓ®#T Sx{ￊà *b§«ª zpIx Ï$]Ïêﾘ×N>

**Comment by darklord1807** [comment]
Can this be fixed?

**Comment by milankragujevic** [comment]
I had success using youtube-dl to fetch the stream URL then proxying the URL with Express on NodeJS... You should try that, if I have time I'll make a demo...

**Comment by Sarfroz** [comment]
i tried but the stream data again in encode form 
/stream/gH)Y1gY"1SgY`10y2Yg3x)1hg0Y01'g:Y41eg&Yr1ug>Y%1GgYV1Ng1Y1LguY1ag9Y1fg{Y1`g/Y-1dg{Y61xg1Y1GgxY1dgY!1Xg{Y1GgY<1DgxY1hg!Y1NgyY1|g;Y1^g{Y

**Comment by Majestcx** [comment]
Now stream link is shared after "Play" button click, so you have to simulate a click

**Comment by Majestcx** [comment]
It's a problem of PhantomJS

**Comment by Majestcx** [comment]
Now OpenLoad check if PhantomJS is used and block the "streamurl" generation.
I have tried to change User-Agent but nothing has changed

**Comment by Majestcx** [comment]
They use something like that: https://blog.shapesecurity.com/2015/01/22/detecting-phantomjs-based-visitors/

**Comment by Majestcx** [comment]
If anyone is interested, this is all the JS code set that the page performs. Maybe it's possible to get the generator of "streamurl":

https://openload.co/f/U_ZBOUcdKAg/OpenLoad_Scripts_JS.js.gz

**Comment by jhaylee2016** [comment]
i see you updated the code is it working now...

**Comment by Sarfroz** [comment]
nope it will not work, see the js code you will find following
`window.callPhantom||/Phantom/.test(navigator.userAgent)||window.__phantomas||pt()||window.domAutomation||window.webdriver||document.documentElement.getAttribute('webdriver'))`

**Comment by Majestcx** [comment]
@jhaylee2016 could you send an image?

**Comment by jhaylee2016** [comment]
no i mean this script was updated 3 days ago..im asking if it is working??..sorry i have no time to test it..but on my part ive taken some research and test about how to get the streamable links..i saw some of the code on openload when I open a view source the direct link was encoded with aaencode but you can decode it w/ aadecode...its a javascript obfuscator...

**Comment by Majestcx** [comment]
@jhaylee2016,I have already decoded it, but in the js there is a script that check if browser is a phantom like, and block the execution

**Comment by jhaylee2016** [comment]
@Majestcx so we don't have any solution to block also the script that check if it is a phantom...maybee....

**Comment by Majestcx** [comment]
@jhaylee2016 it's encoded and I have no time to try an replace code or so...
